pymc_marketing/mmm/utils.py

```diff
@@ -22,7 +22,7 @@
 import xarray as xr
 from scipy.optimize import curve_fit, minimize_scalar
 
-from pymc_marketing.mmm.transformers import michaelis_menten
+from pymc_marketing.mmm.transformers import michaelis_menten_function
 
 
 def estimate_menten_parameters(
```

Comment:
Where is this still used? I thought there was a generalization of this? @cetagostini

Reply:
Created an issue here: #1055